src/styles/blocks/card.scss

```diff
@@ -0,0 +1,101 @@
+.card {
+  width: 200px;
+  background-color: #fff;
+
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: flex-start;
+  align-items: center;
+  box-sizing: border-box;
```

Comment:
it is a good practice to add `box-sizing: border-box;` on all elements using `*`